btr-api/Dockerfile

```diff
@@ -83,7 +83,7 @@ USER web
 
 # The following stage is only for production:
 FROM development_build AS production_build
-COPY --chown=web:web . /code
+COPY --chown=web:web .. /code
```

Comment:
Can someone verify this is correct, or it got messed up during moving process.

Reply:
Hmm I think this shouldn't have changed -- did you try building it?